### DISCUSSION TITLE
Manual process on top of git tags is a chore

### DISCUSSION THREAD

**Discussion by kornelski** [community/community#5986] [Other Feature Feedback, Questions, & Ideas] [answered]
I use command-line git for tagging releases, but these don't count as proper releases in GitHub UI. This confuses my users. The new page makes it even worse by hiding the tags entirely. "There aren’t any releases here" may mislead users that the project has no releases, rather than it's just not using this part of the service.

GitHub requires me to manually open a page in my browser and click things. I suppose I could also write an API client to automatically create releases, but that also feels like overkill to save a couple of clicks I do once in a while.

I would strongly prefer to have releases auto-generated based on a semver-like pattern of tags (e.g. `v1.0.5-beta`).

I keep forgetting to visit GitHub releases page to make a new release there, and my users are confused why my "latest" (marked by GitHub) release is outdated, while in fact I'm making releases regularly, but using standard git workflow instead.

Please, support git-only workflow without a need to manually update via Web GUI.

**Comment by Gakk** [comment]
I agree, the tags should be shown in between releases.

On a sidenote, we use workflows to automatically [generate GitHub Release on tag creation](https://github.com/softprops/action-gh-release), so this is an option to always have a "proper" release. But anyway sometimes a tag is created but it ended up being an internal release, and therefore only is visible with a tag on GitHub and no "proper" release.

**Comment by MylesBorins** [comment, marked as answer]
Here is an example workflow for automating the process of creating a release whenever a tag is pushed

https://github.com/MylesBorins/node-osc/blob/main/.github/workflows/create-release.yml

We opted to removing the tags from the release page as we had received quite a bit of feedback that showing all tags that didn't have releases created a significant amount of noise.

Check out this thread for more details

https://github.community/t/tag-without-release/857

We are definitely interested in trying to make this process even easier for folks but we want to ensure that we have close to 100% accuracy before we automate anything by default.

We could explore making an option in the `release.yml` to opt into a fairly basic publish by default, I'll have to think on it and talk to a few folks

**Comment by kornelski** [comment]
Regarding tags, it was a problem because you were showing **all** tags, rather than **release tags**. I would prefer if the releases page shown only tags that match a pattern I use for release tags, e.g. `v{semver}`.
